### ISSUE TITLE
Conv2dOp Conversion Assertion Failure: Unequal Padding Top and Bottom Not Supported in TTNN

### ISSUE THREAD

**Issue by kamalrajkannan78** [tenstorrent/tt-mlir#1373]
will add repro steps soon.

**Comment by jserbedzijaTT** [comment]
I’ve opened an issue in Metal requesting support for padding in all directions: https://github.com/tenstorrent/tt-metal/issues/17656.

**Comment by kamalrajkannan78** [comment]
> ```
> %65 = "ttir.conv2d"(%63, %arg15, %arg16, %64) <{dilation_height = 1 : si32, dilation_width = 1 : si32, groups = 48 : si32, operand_constraints = [#tt.operand_constraint<dram|l1|scalar|tile|none|interleaved|single_bank|height_sharded|width_sharded|block_sharded|any_layout|any_device|any_device_tile|l1_block_sharded>, #tt.operand_constraint<dram|l1|scalar|tile|none|interleaved|single_bank|height_sharded|width_sharded|block_sharded|any_layout|any_device|any_device_tile|l1_block_sharded>, #tt.operand_constraint<dram|l1|scalar|tile|none|interleaved|single_bank|height_sharded|width_sharded|block_sharded|any_layout|any_device|any_device_tile|l1_block_sharded>, #tt.operand_constraint<dram|l1|scalar|tile|none|interleaved|single_bank|height_sharded|width_sharded|block_sharded|any_layout|any_device|any_device_tile|l1_block_sharded>], padding_bottom = 2 : si32, padding_left = 1 : si32, padding_right = 2 : si32, padding_top = 1 : si32, stride_height = 2 : si32, stride_width = 2 : si32}> {channel_last = 1 : si32} : (tensor<1x128x128x48xf32>, tensor<48x1x5x5xf32>, tensor<1x48x64x64xf32>, tensor<1x64x64x48xf32>) -> tensor<1x64x64x48xf32>
> ```
> 
> Looking at the generated TTIR, this conv2d op has: `padding_bottom = 2 : si32, padding_left = 1 : si32, padding_right = 2 : si32, padding_top = 1 : si32`
> 
> TTNN currently supports only two padding values, where the padding-top must be equal to the padding-bottom, and the padding-right must be equal to the padding-left.
> 
> [@kamalrajkannan78](https://github.com/kamalrajkannan78) are these padding values expected from this model? If it is, then we need to request a feature request from the TTNN team to implement this.
> 
> Assigning this to [@jserbedzijaTT](https://github.com/jserbedzijaTT) to create an issue on the metal side.

Yes @sdjordjevicTT  , these padding values  are expected from model!

**Comment by nvukobratTT** [comment]
> [@nvukobratTT](https://github.com/nvukobratTT) [@kamalrajkannan78](https://github.com/kamalrajkannan78) Can I close this issue since you said you will handle it from the FE side?

We'll handle it on the frontend to better fit PyTorch. We can close it for now
